stacks/apps/feeder.yml

```diff
@@ -251,47 +251,49 @@ Resources:
       Policies:
         - PolicyDocument:
             Statement:
-              - Action: sqs:*
+              - Action:
+                  - sqs:ChangeMessageVisibility
+                  - sqs:Get*
+                  - sqs:List*
+                  - sqs:ReceiveMessage
+                  - sqs:SendMessage
                 Effect: Allow
                 Resource:
-                  - !Sub arn:${AWS::Partition}:sqs:${AWS::Region}:${AWS::AccountId}:*_feeder_*
                   - !GetAtt DefaultJobQueue.Arn
-                  - !GetAtt DefaultJobDeadletterQueue.Arn
                   - !GetAtt FixerCallbackQueue.Arn
-                  - !GetAtt FixerCallbackDeadletterQueue.Arn
```

Comment:
Yeah, I guess shoryuken never directly interacts with the DLQ - it's AWS only.  +1.